article/views.py

```diff
@@ -0,0 +1,181 @@
+from django.views.generic import TemplateView
+
+
+class BaseArticleView(TemplateView):
+    template_name = 'article/base.html'
+
+    def get_context_data(self, *args, **kwargs):
+        return super().get_context_data(
```

Comment:
think you need to move the extra kwarg to the end - think this will fail if there will be any args (as using kwarg, args, kwargs)